pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.parasoft</groupId>
     <artifactId>parasoft-findings-utils</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.0.0</version>
```

Comment:
Make it have the same pattern(x.x.x) with Parasoft Findings plugins. https://docs.parasoft.com/display/FINDS/Release+Notes